src/OpenCVBuildEnv.ts

```diff
@@ -673,12 +674,13 @@ export default class OpenCVBuildEnv implements OpenCVBuildEnvParamsBool, OpenCVB
     get optHash(): string {
         if (this.hash)
             return this.hash;
-        let optArgs = this.getCongiguredCmakeFlags().join(' ');
+        let optArgs = this.getConfiguredCmakeFlags().join(' ');
         if (this.buildWithCuda) optArgs += 'cuda'
         if (this.isWithoutContrib) optArgs += 'noContrib'
         if (optArgs) {
             optArgs = '-' + crypto.createHash('md5').update(optArgs).digest('hex').substring(0, 5);
         }
+        this.hash = optArgs;
```

Comment:
I need to double-check that. I may have intentionally omitted this set hash.

The hash can change during the configuration process, and I want to set it on purpose explicitly.